tests/snap_test.go

```diff
@@ -0,0 +1,203 @@
+package tests
+
+import (
+	"context"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/canonical/matter-snap-testing/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	// enviroment variables
+	SpecificGpioChip = "GPIO_CHIP"
+	SpecifGpioLine   = "GPIO_LINE"
+)
+
+var (
+	TestedGpioChip = ""
+	TestedGpioLine = ""
+)
+
+func init() {
+	TestedGpioChip = os.Getenv(SpecificGpioChip)
+	TestedGpioLine = os.Getenv(SpecifGpioLine)
+}
```

Comment:
Only used once inside the setupGpio function. It could be moved to there.

Reply:
Okay, fixed, thank you!